src/components/study-pane.jsx

```diff
@@ -260,7 +96,18 @@ const StudyPane = ({ studyUuid, currentNode, setErrorMessage, ...props }) => {
                     display: props.view === StudyView.MAP ? null : 'none',
                 }}
             >
-                {renderMapView()}
+                <MapView
```

Comment:
Even if it's obviously a good idea to create this component and move it in a proper file, it makes the review more difficult ! It's better to do a separate PR for those refactoring achievements.

Reply:
Ok keep this refacto in this PR